schema/rdl_schema_0.1.json

```
@@ -40,10 +41,12 @@
         "Loss"
       ]
     },
-    "organization": {
-      "type": "string",
-      "description": "Name of organization that published the dataset",
-      "minLength": 1
+    "publisher": {
+      "title": "Publisher",
+      "type": "object",
+      "description": "The entity responsible for making the dataset available.",
+      "$ref": "#/$defs/Entity",
+      "minProperties": 2
```

`minProperties` is already set on `Entity`.

```suggestion
      "$ref": "#/$defs/Entity"
```